services/src/main/java/org/keycloak/protocol/oidc/OIDCWellKnownProvider.java

```diff
@@ -253,6 +258,18 @@ public void close() {
         return getSupportedAlgorithms(ContentEncryptionProvider.class, false);
     }
 
+    private List<String> getAcrValuesSupported(RealmModel realm) {
+        // Values explicitly set on the realm mapping
+        Map<String, Integer> realmAcrLoaMap = AcrUtils.getAcrLoaMap(realm);
+        List<String> result = new ArrayList<>(realmAcrLoaMap.keySet());
+
+        // Add LoA levels configured in authentication flow in addition to the realm values
+        result.addAll(AuthenticatorUtil.getLoAConfiguredInRealmBrowserFlow(realm)
```

Comment:
How about removing duplicates?
The current keycloak accepts the following `acr-to-loa` mapping.
ACR : 0 -> LOA : 0
ACR : 1 -> LOA : 0
I think it is a very subtle problem so that we might need not to consider it too much.